### PR TITLE
f-enhancement/gh-358-request-number-indicator

### PR DESCRIPTION
**Pull request by aberonja**
Added request number indicator.

GH issue: #358

### CODE REVIEW

File: src/main/frontend/src/components/navigation/Sidebar.js [hunk 1]
```diff
@@ -32,7 +38,27 @@ const LinkWrapper = styled.div`
...
+  position: relative;
+  &:after {
+    content: ${props =>
+      props.requestsTotal ? `'${props.requestsTotal}'` : ''};
+    color: red;
+    position: absolute;
+    top: ${props => (props.requestsTotal > 99 ? '-7px' : '-5px')};
+    left: ${props => (props.requestsTotal > 99 ? '-7px' : '-5px')};
+    width: ${props => (props.requestsTotal > 99 ? '25px' : '20px')};
+    height: ${props => (props.requestsTotal > 99 ? '25px' : '20px')};;
```
Comment (BraneOpacic21) [inline src/main/frontend/src/components/navigation/Sidebar.js]: Maybe extract this to a function? .. or no need? @iscekic

File: src/main/frontend/src/components/navigation/Sidebar.js [hunk 2]
```diff
@@ -50,7 +76,12 @@ const navLinks = (isMinimized, searchedEmployee, location) =>
       icon: LeaveIcon,
       searchedEmployee
     },
-    { url: '/approvals', name: 'Approvals', icon: LeaveHistoryIcon },
+    {
+      url: '/approvals',
+      name: 'Approvals',
+      icon: LeaveHistoryIcon,
+      requestsTotal: requestsTotal
```
Comment (BraneOpacic21) [inline src/main/frontend/src/components/navigation/Sidebar.js]: `requestsTotal`

File: src/main/frontend/src/components/navigation/Sidebar.js [hunk 3]
```diff
@@ -132,7 +169,8 @@ class Sidebar extends Component {
             {navLinks(
               this.state.isMinimized,
               this.props.searchedEmployee,
-              this.props.router.location
+              this.props.router.location,
+              this.props.requestsTotal
```
Comment (BraneOpacic21) [inline src/main/frontend/src/components/navigation/Sidebar.js]: ES6 destructuring could be useful here

File: src/main/frontend/src/store/reducers/approvalsReducer.js [hunk 4]
```diff
@@ -2,11 +2,24 @@ import { handleActions } from 'redux-actions';
...
+  total: 0
+};
 
 const actionHandlers = {
-  [receiveRequestsForApproval]: (state, action) => action.payload,
-  [navigateOut]: () => null
+  [receiveRequestsForApproval]: (state, action) => {
+    return {
+      requests: action.payload,
+      total: action.payload.length
```
Comment (BraneOpacic21) [inline src/main/frontend/src/store/reducers/approvalsReducer.js]: Destructure `{payload}` from action to avoid repeating yourself.

Comment (aberonja) [reply src/main/frontend/src/store/reducers/approvalsReducer.js]: I would skip it here, since there are only two values 🐑

Comment (BraneOpacic21) [reply src/main/frontend/src/store/reducers/approvalsReducer.js]: Cool, reminder for future then! :sheep 💻

File: src/main/frontend/src/components/navigation/Sidebar.js (hunk 1 above)
Comment (aberonja) [reply src/main/frontend/src/components/navigation/Sidebar.js]: Master @iscekic, what do you say about this? 🐑 😋

**Pull request merged**